src/main/kotlin/com/cognifide/gradle/aem/environment/docker/domain/HttpdContainer.kt

```diff
@@ -12,7 +12,7 @@ class HttpdContainer(environment: Environment) {
 
     val container = DockerContainer(aem, aem.props.string("environment.httpdContainer.containerName") ?: "aem_httpd")
 
-    var awaitRetry = aem.retry { afterSecond(aem.props.long("environment.httpdContainer.awaitRetry") ?: 10) }
+    var awaitRetry = aem.retry { afterSecond(aem.props.long("environment.httpdContainer.awaitRetry") ?: 30) }
```

Comment:
on first `environmentUp` it was failing. on second not. after changing to bigger value works in both cases.